### ISSUE TITLE
Error: 'coef.cv.glmnet' is not an exported object from 'namespace:glmnet'

### ISSUE THREAD

**Issue by dweemx** [ebi-gene-expression-group/garnett-cli#10]
Hi,
I'm trying to run `garnett_get_features_genes.R` but got the following error:
```
garnett_get_feature_genes.R \
   --classifier-object Garnett_Classifier.Rds \
   --database org.Dm.eg.db \
   --output-path Garnett_Classifier_Feature_Genes.txt
```
I ran also `garnett_classify_cells.R`:
```
garnett_classify_cells.R \
   --cds-object Monocle_CDS.Rds \
   --classifier-object Garnett_Classifier.Rds \
   --database org.Dm.eg.db \
   --cds-gene-id-type SYMBOL \
   --cds-output-obj Monocle_CDS_Classified.Rds
```

and got the same type of error:
```
<simpleError: 'coef.cv.glmnet' is not an exported object from 'namespace:glmnet'>
```

`garnett-cli` was installed in a conda environment (as recommended)

N.B.: The `get_features_genes` works with `garnett_0.1.14` (in the R conda environment, there is `garnett_0.2.8`)

**Comment by a-solovyev12** [comment]
Hi @dweemx , 

This is likely caused by dependency conflicts with `r-glmnet`. Downgrading to version `2.0_18` should solve it. Please let us know if you have any further issues. 

Andrey

**Comment by dweemx** [comment]
Hi @a-solovyev12,

Downgrading to `2.0_18` solves the issue. 

Are you planning to update and fix the version `r-glmnet==2.0_18` in the `garnett-cli` bioconda package ?

**Comment by a-solovyev12** [comment]
@dweemx yes, the package will be updated soon.
